znap-cli/src/utils/mod.rs

```diff
@@ -75,14 +75,16 @@ pub fn start_server_blocking(
 
 pub fn start_server(
     name: &str,
-    identity: &str,
+    identity: Option<&str>,
     address: Option<&str>,
     port: Option<&u16>,
     protocol: Option<&str>,
 ) -> Child {
     let mut env_vars: HashMap<&str, String> = HashMap::new();
 
-    env_vars.insert("IDENTITY_KEYPAIR_PATH", identity.to_owned());
+    if let Some(i) = identity {
```

Comment:
It there's always a presence of an identity in the toml, this should set the value based on the precedence order: Env Variable -> Toml Config